index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <section class ="content">
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
+        </a>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </nav>
+    </header>
+    <div id="greeting">
```

Comment:
- **[OPTIONAL]** This section seems too close to the navbar. Adding more spacing to it will help the Intro page to fill the remaining screen space. It would be great if you could add space to match it with the Figma template. 👍🏽 

<img width="371" alt="Screenshot 2023-01-19 at 4 43 10 PM" src="https://user-images.githubusercontent.com/24832846/213428114-ef647040-897b-4352-8210-4981623cc618.png">

Figma Template
<img width="353" alt="Screenshot 2023-01-19 at 4 43 25 PM" src="https://user-images.githubusercontent.com/24832846/213428181-aa05f3cd-c378-472d-9347-c2d5b0580a4c.png">